backends/tofino/compiler_interfaces/.python-version

```diff
@@ -0,0 +1 @@
+3.12
```

Comment:
Is adding this .python-version intentional?  What uses it?

Reply:
Yes, this picks the default Python version for this particular project. It's optional, I might remove it once I clean up these particular files.